packages/config/config/devices/0x027a/zse29.json

```diff
@@ -119,7 +119,15 @@
 		"5": {
 			"$if": "firmwareVersion >= 1.03",
 			"label": "Motion Re-trigger Time",
-			"description": "Delay before the sensor reports no motion after detecting the last motion activity.",
+			"description": [
+				{
+					"$if": "firmwareVersion < 2.00",
```

Comment:
That doesn't work, you'll have to repeat the entire parameter if you want to change the description.
Also, firmware versions have no leading zeros, even if manufacturers call them that way.

Reply:
I'll change the firmware reference.  I think you're looking at an outdated version.  I've just limited all of these parameters to firmware >= 2, as mentioned above.